app/_includes/nav-v2.html

```diff
@@ -6,16 +6,33 @@
         alt="Kong Logo"
       />
     </a>
+    {% if page.layout != "landing-page" %}
+    <div class="search-input-wrapper">
+      <img src="/assets/images/landing-page/search.svg" alt="search">
+      <input id="getkong-algolia-search-input" type="text" placeholder="Search...">
```

Comment:
```suggestion
      <input id="getkong-algolia-search-input" type="text" placeholder="Search the docs...">
```
To avoid confusion that they might be searching all of Kong's learning offerings (support, demos, etc), better to change this. Maybe shorten to just "Search..." on mobile?

Can we also double the length of the search bar on full-size desktops?